onnx/defs/function.h

```diff
@@ -19,4 +21,80 @@ void FunctionExpandHelper(
     const FunctionProto& func,
     GraphProto& g,
     const std::string& node_prefix = "");
+
+class FunctionBodyHelper {
+ public:
+  struct AttributeProtoWrapper {
+    AttributeProto proto;
+
+    AttributeProtoWrapper() {}
+
+    AttributeProtoWrapper(const AttributeProto& attr_prot) {
```

Comment:
looks to me this wrapper is not that useful, but creating one more copy. I may miss sth though.

Reply:
This class serve as a syntactic sugar for easily define an attribute, so that user simply write following without worrying about how to compose a AttributeProto. 

{"attribute", 1.0f}
{"attribute", int64_t(1)}
{"attribute", "string_value"}
{MakeRefAttribute("axes", AttributeProto::INTS)}}